github/git_blobs_test.go

```diff
@@ -22,6 +22,11 @@ func TestGitService_GetBlob(t *testing.T) {
 		if m := "GET"; m != r.Method {
 			t.Errorf("Request method = %v, want %v", r.Method, m)
 		}
+
+		if m := mediaTypeGraphQLNodeIDPreview; m != r.Header.Get("Accept") {
+			t.Errorf("Request accept header = %v, want %v", r.Header.Get("Accept"), m)
+		}
```

Comment:
@shurcooL  While working on this file, I noticed that in [git_blobs_test.go](https://github.com/google/go-github/blob/master/github/git_blobs_test.go#L22) and [git_trees_test.go](https://github.com/google/go-github/blob/master/github/git_trees_test.go#L22) we have used:
> 		if m := "GET"; m != r.Method {
>			t.Errorf("Request method = %v, want %v", r.Method, m)
>              }
which is tested differently from the rest of the files for the header where we have done it like :
>		testMethod(t, r, "GET")
 Is their any specific reason for doing so? I implemented the test on the **accept header** to maintain the consistency with the respective files. Let me know your thoughts.

Reply:
@anubhakushwaha IMO, the inconsistency in `git_trees_test.go` and `git_blobs_test.go` is by accident rather than having any reasoning behind it. I'll suggest you to use `testHeader` in your PR to make it consistent with the entire package.

We should instead fix the inconsistency to avoid confusions like this one in future.

Reply:
@sahildua2305  Thanks for the suggestion, will update the PR.

Reply:
Are you still planning to do this? Just checking.